integration_test/perf_driver.dart

```diff
@@ -0,0 +1,24 @@
+// This file is derived from BSD licensed example code in
```

Comment:
Let's put this part (about being derived from some other code, and the license of that code) in the commit message rather than an in-tree comment. We've now modified this file a fair bit, and may modify it more in the future; so that's the form that will best avoid getting outdated with such changes.

The link to the docs is still good as context for someone editing this file in the future.